docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+
+services:
+  grafana:
+    image: grafana/grafana-oss:9.1.0-ubuntu
+    volumes:
+      - .:/var/lib/grafana/plugins/grafana-checkmk-datasource
+    environment:
+      GF_ANALYTICS_REPORTING_ENABLED: 'false'
+      GF_ANALYTICS_CHECK_FOR_UPDATES: 'false'
+      GF_SECURITY_ADMIN_USER: admin
+      GF_SECURITY_ADMIN_PASSWORD: password
+      GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS: tribe-29-checkmk-datasource
+    ports:
+      - '3000:3000'
+    depends_on:
+      - checkmk
+  checkmk:
+    image: checkmk/check-mk-free:2.1.0p9
+    ports:
+      - '12345:5000'
```

Comment:
this is very subjective, but i would prefer `5001` as an external port. if you like `12345` i'm also ok with that.